### DISCUSSION TITLE
Dropdown menu of the nz-select is not scrolling along with the drawer—it remains fixed in place.

### DISCUSSION THREAD

**Discussion by askarmus** [ng-alain/ng-alain#2535] [General]
The nz-select dropdown menu does not scroll along with its parent container when placed inside a scrollable drawer. Instead, the dropdown remains fixed in its initial position, making it misaligned when the drawer content is scrolled. This behavior impacts usability, especially in scenarios with longer forms or lists inside a drawer.

![image](https://github.com/user-attachments/assets/9c13b15b-7630-4e7a-aa37-31d6c2ec00f4)

Above issue can reproduced from official theme
https://ep.ng-alain.com/yun/browser/#/stat/service

**Comment by cipchk** [comment]
This is a historical issue with NG-ZORRO, but it has been fixed in `18.2.0` (https://github.com/NG-ZORRO/ng-zorro-antd/pull/8804). However, the DEMO here is still using NG-ZORRO version 18.0.0. I will update it in the future.
